src/sentry/api/serializers/models/group.py

```diff
@@ -195,8 +197,12 @@ def serialize(self, obj, attrs, user):
             status_details['autoResolved'] = True
         if status == GroupStatus.RESOLVED:
             status_label = 'resolved'
-            if attrs['pending_resolution']:
-                status_details['inRelease'] = attrs['pending_resolution']
+            if attrs['resolution']:
+                res_type, res_version = attrs['resolution']
+                if res_type in (GroupResolution.Type.in_next_release, None):
```

Comment:
I mean, I didn't see it as confusing since it's still just simply a value you're working with. And as I said, the space doesn't matter here, but on high millions of rows, a 32 bit integer begins to add up.